src/Wt/WGoogleMap

```diff
@@ -152,7 +152,7 @@ public:
 
   /*! \brief Adds a icon marker overlay to the map.
    */
-  void addIconMarker(const Coordinate &pos, const std::string& iconURL);
+  void addIconMarker(const Coordinate &pos, const std::string& iconURL, std::string toolTip = NULL);
```

Comment:
Since tooltip is text that should be translatable, the following signature is more appropriate:
void addIconMarker(const Coordinate &pos, const std::string& iconURL, const Wt::WString toolTip = Wt::WString());